### ISSUE TITLE
See one of your lists on the map

### ISSUE THREAD

**Issue by stavzi222** [theCrag/website#4041]
One common usage case of a list at least for me is to create a list of routes I want to climb.

However when looking at the list I cannot plan my trip before I have to click on all the routes to remember where they are exactly and if the weather + time is permitting me to pursue them.

This makes the list kind of useless.

However if I could simply see the routes on the list on a map similar to the way the map works today only this time it will show only climbs from my list then I can easily decide what from my list I can pursue.

I think this is a much needed addition to lists to make them much more usable and diminish their current "bookmarks which I never open again" nature.

**Comment by killakalle** [comment]
@stavzi222 

I agree that the current list view has not so much practical use. Though, when you click on the route count, you get to a view that shows the routes according to their crag and sector.

![image](https://user-images.githubusercontent.com/30682796/159793460-443a738a-b442-4454-9f9a-bee1c0d8753d.png)

![image](https://user-images.githubusercontent.com/30682796/159793537-594dd391-2d0a-4329-970d-bf5bc337c413.png)

**Comment by Matthewstuartperkins** [comment]
**Current unsatisfying workaround:**
I have a list off all my climbing goals. I manually create lists for each areas that can be accessed in a single day trip.  
When planning trip I choose a goal route then look at specific area list for nearby climbs that I'm also interested in.  

**New Feature:**
A profile map that shows a specific list of climbs. 

**Use case:**
You have created a list of all your climbing goals. With these displayed on map you can now easily see and choose crags based on only the routes you are excited to climb.

**How it could look:**
![Goals Map](https://user-images.githubusercontent.com/37822688/201227506-eba0bfd3-5384-4f01-918c-3ce22a22c8e1.png)

**Comment by tuxfy** [comment]
This is also for me a highly missed feature. Not possible with 27crags, vertical life or thecrag. But at every climbing trip this is the first dicussion. 

Which climbs/boulders are interesting, where can I find them, how to try them in a efficient order. Espacially on short weekend trips a most have to not loose time for walking around.

**Comment by nicHoch** [comment]
this was rolled out last release
